src/Stratis.Bitcoin.Features.Miner/Staking/PosMinting.cs

```diff
@@ -432,17 +432,20 @@ internal async Task<List<UtxoStakeDescription>> GetUtxoStakeDescriptionsAsync(Wa
                 .GetSpendableTransactionsInWalletForStaking(walletSecret.WalletName, 1).ToList();
 
             FetchCoinsResponse fetchedCoinSet = await this.coinView.FetchCoinsAsync(spendableTransactions.Select(t => t.Transaction.Id).Distinct().ToArray(), cancellationToken).ConfigureAwait(false);
+            Dictionary<uint256, UnspentOutputs> utxoByTransaction = fetchedCoinSet.UnspentOutputs.ToDictionary(utxo => utxo.TransactionId, utxo => utxo);
```

Comment:
Can you not move this value 
`utxo.Value < MinimumStakingCoinValue`


up to here?
`pendableTransactions.Where(w -> ).Select(t => t.Transaction.Id)`